packages/webapp/src/config.ts

```diff
@@ -73,6 +73,11 @@ export const chainConfig = {
     rpcEndpoints: [rpcEndpoint],
 };
 
+
+export const ipfsUrl = (ipfsHash: string) => `${ipfsBaseUrl}/${ipfsHash}`;
+export const explorerAccountUrl = (accountName: string) =>
+    `${blockExplorerAccountBaseUrl}/${accountName}`;
+
```

Comment:
I'd like to keep this file clean with only the config settings. let's move this to _app/utils/config-helpers.ts or something. (I'm working on it!)